sdk/react/ooyalaSkinPanelRenderer.js

```diff
@@ -220,9 +220,16 @@ OoyalaSkinPanelRenderer.prototype.renderAudioAndCCSelectionPanel = function() {
 };
 
 OoyalaSkinPanelRenderer.prototype.renderPlaybackSpeedPanel = function() {
-  return (
+  let playbackSpeedRates;
+
+  if (this.skin.props.playbackSpeed && Array.isArray(this.skin.props.playbackSpeed.options)) {
+    playbackSpeedRates = this.skin.props.playbackSpeed.options;
+  } else {
+    playbackSpeedRates = [];
```

Comment:
you can simplify this logic. The else can be omitted if you initialize the variable as an empty array. E.g.
```
let playbackSpeedRates = [];
if (this.skin.props.playbackSpeed && Array.isArray(this.skin.props.playbackSpeed.options)) {
    playbackSpeedRates = this.skin.props.playbackSpeed.options;
  }
```